central/complianceoperator/v2/report/manager/generator/report_gen_impl.go

```diff
@@ -100,9 +100,9 @@ func (rg *complianceReportGeneratorImpl) ProcessReportRequest(req *report.Reques
 
 	if snapshot != nil {
 		snapshot.GetReportStatus().RunState = storage.ComplianceOperatorReportStatus_GENERATED
-		if len(req.FailedClusters) > 0 {
+		if req.FailedClusters > 0 {
```

Comment:
hmm, are you sure we want to remove the `len()` from here?
Or maybe the `req.FailedClusters` should be now called `req.NumFailedClusters`?

Reply:
I found it - let's rename the field to `NumFailedClusters`